src/antsibull_core/logging.py

```diff
@@ -243,4 +246,75 @@ def initialize_app_logging() -> None:
     twiggy.quick_setup(min_level=_level)
 
 
-__all__ = ("log", "initialize_app_logging")
+class Logger(metaclass=abc.ABCMeta):
+    @abc.abstractmethod
+    def debug(self, format_spec: str, *args, **kwargs) -> None:
+        pass
+
+    @abc.abstractmethod
+    def info(self, format_spec: str, *args, **kwargs) -> None:
+        pass
+
+    @abc.abstractmethod
+    def notice(self, format_spec: str, *args, **kwargs) -> None:
```

Comment:
`notice()` seems unique to twiggy; I guess we can just have this method call `info()` with the same arguments when/if we migrate.